de_DE/LC_MESSAGES/discordbot.po

```diff
@@ -49,47 +49,47 @@ msgstr "%d Suchergebnisse:"
 
 # SongName by BandName
 msgid "MUSICMATCH_TITLE_BY_ARTIST"
-msgstr "%s von %s"
+msgstr "%s - %s"
 
 msgid "PATRON_TICKET_SUCCESS"
-msgstr "You received a %s Sudomemo Patron Ticket!"
+msgstr "Du hast ein %s Sudomemo Patron Ticket erhalten."
 
 msgid "REJECT_ACCOUNT_ALREADY_LINKED"
-msgstr "Du hast dein Sudomemo-Konto bereits verbunden!"
+msgstr "Du bist bereits in Sudomemo angemeldet."
 
 msgid "REJECT_BOOST_TICKET_COOLDOWN"
-msgstr "Du hast bereits ein Nitro Booster-Ticket in den letzten 30 Tagen erhalten! (%s)"
+msgstr "Du hast bereits ein Nitro Booster-Ticket in den letzten 30 Tagen erhalten, (%s)."
 
 msgid "REJECT_FSID_ENOENT"
-msgstr "Für diese Flipnote Studio ID wurde kein Benutzer gefunden."
+msgstr "Diese Flipnote Studio ID (FSID) existert nicht."
 
 msgid "REJECT_GENERIC_ERROR"
-msgstr "Bei dieser Aktion trat ein Problem auf. Sorry!"
+msgstr "Ein Fehler ist aufgetreten. Bitte versuche es in 5 Minuten noch einmal."
 
 msgid "REJECT_INVALID_FSID"
-msgstr "Dies ist keine gültige Flipnote Studio ID!"
+msgstr "Ungültige Flipnote Studio ID (FSID)."
 
 msgid "REJECT_INVALID_LINK_TOKEN"
-msgstr "Dies ist kein gültiger Verbindungs-Token! Bitte besuche %s, um loszulegen!"
+msgstr "Dieser Zugangscode ist ungültig. Bitte besuche %s um einen neuen Code zu erhalten."
 
 msgid "REJECT_LINK_TOKEN_ENOENT"
-msgstr "Der vorgegebene Verbindungs-Token ist entweder abgelaufen oder existiert nicht."
+msgstr "Dieser Zugangscode ist abgelaufen. Bitte besuche %s um einen neuen Code zu erhalten."
 
 msgid "REJECT_NOT_NITRO_BOOSTER"
-msgstr "Du musst ein Sudomemo Nitro Booster sein, um dies zu nutzen!"
+msgstr "Du musst ein Sudomemo Nitro Booster sein, um dies nutzen zu können."
 
 msgid "REJECT_NOT_PATRON"
-msgstr "You'll need to be a Sudomemo Patron to use this!"
+msgstr "Du musst ein Sudomemo Patron sein, um dies nutzen zu können."
 
 msgid "REJECT_PATRON_TICKET_COOLDOWN"
-msgstr "You've already claimed your Patron ticket for this month!"
+msgstr "Du hast bereits ein Patron-Ticket in den letzten 30 Tagen erhalten, (%s)."
 
 msgid "REJECT_USER_ENOENT"
-msgstr "Dieser Nutzer konnte nicht auf Sudomemo gefunden werden."
+msgstr "Dieser Nutzer existiert nicht."
```

Comment:
This is too generic, the message specifies that this **Sudomemo** user could not be found, which may not be obvious in context.